src/pykoala/spectra/onedspec.py

```diff
@@ -0,0 +1,3015 @@
+# 6312.10     [SIII]  -0.264   30.0 18.0    5.0   20.0
+# 6363.78       [OI]  -0.271   20.0  4.0    5.0   30.0
+# 6548.03      [NII]  -0.296   45.0 15.0   55.0   75.0
+# 6562.82         Ha  -0.298   50.0 25.0   35.0   60.0
+# 6583.41      [NII]  -0.300   62.0 42.0    7.0   35.0
+# 6678.15        HeI  -0.313   20.0  6.0    6.0   20.0
+# 6716.47      [SII]  -0.318   40.0 15.0   22.0   45.0
+# 6730.85      [SII]  -0.320   50.0 30.0    7.0   35.0
+# 7065.28        HeI  -0.364   30.0  7.0    7.0   30.0
+# 7135.78    [ArIII]  -0.374   25.0  6.0    6.0   25.0
+# 7318.39      [OII]  -0.398   30.0  6.0   20.0   45.0
+# 7329.66      [OII]  -0.400   40.0 16.0   10.0   35.0
+# 7751.10    [ArIII]  -0.455   30.0 15.0   15.0   30.0
+# 9068.90    [S-III]  -0.594   30.0 15.0   15.0   30.0 
+
+# EXTRA
+#    8750.47   -0.568       H-I    P12   8776
+#    8862.79   -0.578       H-I    P11   8888
+#    9014.91   -0.590       H-I    P10   9041
+
+
+# NOTE: 18 lines for BLUE (580V) and 18 lines for RED (385R), if this changes careful with emission line detection!!!
+                         
+el_list_no_z = [3726.03, 3728.82,  # 3727.0, #
+                3868.75, 3889.05,  3964.73, 3967.46, 3970.07,  #3966.1 #
+                4068.60 , #4076.35 ,
+                4101.74, 4340.47, 4363.21, 
+                #4437.55, 
+                4471.48, 4658.10,
+                4686.00, 4861.33, 4958.91, 5006.84,
+                5191.82, #5517.71,   
+                #red  #  [OI]   [SIII]  [OI]   Ha+[NII]  HeI    [SII]     HeI   [ArIII]  [OII]  [ArIII]    P12 P11 P10    [SIII]         
+                6300.3, 6312.10, 6363.78, 6548.03, 6562.82, 6583.41, 6678.15, 
+                6716.47, 6730.85, 7065.28, 7135.78, 7318.39, 7329.66, 7751.1, 
+                8750.47, 8862.79, 9014.91,
+                9068.9] 
+
+el_list_names = ["[OII]", "[OII]", 
+                 "[NeIII]",  "H8",  "HeI", "[NeIII]",  "H7", # "HeI+[NeIII]"
+                 "[SII]", #"[SII]",
+                 "Hd","Hg", "[OIII]", 
+                 #"HeI",
+                 "HeI", "[FeIII]",
+                 "HeII", "Hb", "[OIII]", "[OIII]",
+                 "[ArIII]", # "[ClIII]",
+                 #red
+                "[OI]", "[SIII]", "[OI]","[NII]","Ha","[NII]","HeI","[SII]","[SII]","HeI","[ArIII]","[OII]","[OII]","[ArIII]",
+                 "P12", "P11", "P10",
+                 "[SIII]"]
+el_list_fl = [0.322, 0.322,   
+              0.291, 0.286,   0.267, 0.267, 0.266,
+              0.239, #0.237,
+              0.230, 0.157, 0.149,
+              #0.126 ,
+              0.115,  0.058,             
+              0.050,  0.000, -0.032, -0.038,
+              -0.081, #-0.145 ,
+              #red  
+              -0.263, -0.264, -0.271, -0.296,  -0.298,  -0.300, -0.313,  -0.318, -0.320, -0.364, -0.374, -0.398, -0.400, -0.455,
+              -0.568,-0.578 , -0.590,
+              -0.594]
+
+
+
+# EL  dictionary 
+emission_line_dictionary = {}
```

Comment:
I do like this idea.
Would it be more transparent/elegant to implement it through an `EmissionLine` class (perhaps, in the future, a child of `SpectralFeature`)? What do you think?